sdk/communication/azure-communication-chat/src/samples/app/src/androidTest/java/com/azure/android/communication/chat/samples/CreateChatThread.java

```diff
@@ -0,0 +1,111 @@
+package com.azure.android.communication.chat.samples;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import com.azure.android.communication.chat.ChatAsyncClient;
+import com.azure.android.communication.chat.ChatClientBuilder;
+import com.azure.android.communication.chat.models.ChatParticipant;
+import com.azure.android.communication.chat.models.CreateChatThreadOptions;
+import com.azure.android.communication.chat.models.CreateChatThreadResult;
+import com.azure.android.communication.common.CommunicationUserIdentifier;
+import com.azure.android.core.http.HttpClient;
+import com.azure.android.core.http.HttpRequest;
+import com.azure.android.core.http.okhttp.OkHttpAsyncClientProvider;
+import com.azure.autoresttest.BuildConfig;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import java9.util.concurrent.CompletableFuture;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+@RunWith(AndroidJUnit4.class)
+public class CreateChatThread {
+    protected static final String ENDPOINT = BuildConfig.COMMUNICATION_SERVICE_ENDPOINT;
+    protected static final String ACCESS_KEY = BuildConfig.COMMUNICATION_SERVICE_ACCESS_KEY;
+    protected static final String THREAD_MEMBER_1 = BuildConfig.COMMUNICATION_CHAT_THREAD_MEMBER_1;
```

Comment:
It would be better if we use "participant" instead of "member" in naming. Because in both SDK & documents, we use "participant" now.

Reply:
I'll create a work item to do the same renaming in unit test as well.